kv/memberlist/tcp_transport_test.go

```diff
@@ -95,3 +95,10 @@ func TestFinalAdvertiseAddr(t *testing.T) {
 		})
 	}
 }
+
+func TestNonIPsAreRejected(t *testing.T) {
+	cfg := TCPTransportConfig{BindAddrs: flagext.StringSlice{"localhost"}}
+	_, err := NewTCPTransport(cfg, nil, nil)
+	require.Error(t, err)
+	require.Equal(t, err.Error(), `could not parse bind addr "localhost" as IP address`)
```

Comment:
You should be able to simplify this even further:

```suggestion
	require.EqualError(t, err, `could not parse bind addr "localhost" as IP address`)
```

Reply:
Done